Jantimizer/Histograms/Models/Histograms/HistogramEquiDepthVariance.cs

```diff
@@ -42,28 +42,29 @@ protected override void GenerateHistogramFromSorted(List<IComparable> sorted)
                 IComparable startValue = sorted[bStart];
                 IComparable endValue = sorted[bStart];
                 int countValue = 1;
-                int mean = (int)endValue;
-                int variance = 0;
+                double mean = Convert.ToDouble(endValue);
+                double variance = 0;
+                double standardDeviation = 0;
 
                 for (int bIter = bStart + 1; bIter < bStart + Depth && bIter < sorted.Count; bIter++)
                 {
                     countValue++;
                     endValue = sorted[bIter];
-                    mean += (int)endValue;
+                    mean += Convert.ToDouble(endValue);
                 }
                 mean = mean / countValue;
                 for (int bIter = bStart; bIter < bStart + Depth && bIter < sorted.Count; bIter++)
                 {
-                    variance += (int)Math.Pow((int)sorted[bIter] - mean, 2);
+                    variance += Math.Pow(Convert.ToDouble(sorted[bIter]) - (double)mean, 2);
                 }
                 if (countValue > 1 && variance != 0)
-                    variance = (int)Math.Sqrt(variance / countValue - 1);
+                    standardDeviation = Math.Sqrt((double)variance / countValue - 1);
```

Comment:
```suggestion
                    standardDeviation = Math.Sqrt((double)variance / countValue - 1);
```
```suggestion
                    standardDeviation = Math.Sqrt((double)variance / countValue);
```
-1 is for a sample of the population instead of the whole